style.css

```diff
@@ -1,27 +1,221 @@
-body
-{
-background-color: blue;
- }
-
-.navbar
-{
-  display: inline-block;
-  font-size:60px;
-  padding: 10px;
-}
-.icon
-{
-  height: 8%;
-  width: 8%;
-}
-.grid-container
-{
-  display:grid;
-  grid-template-columns: 1fr 1fr;
-  grid-gap: 20px;
-}
-.thumbnail
-{
-  display:block;
-  margin:2%;
+body {
+	box-sizing: content-box;
+	margin: 0;
+	background-color: whitesmoke;
+	font-family:sans-serif;
+}
+
+i {
+	color: #606060;
+}
+
+iframe {
+	width: 100vw;
+	height: 56.25vw; /* 100/56.25 = 560/315 = 1.778 */
```

Comment:
Remove the commented code.

❌ commented piece of codes `/* .nav { color: red;} */`
✔ comments about code /*Navigation*/